src/utils/config.js

```diff
@@ -3,7 +3,7 @@ const path = require('path');
 
 const existsCache = new Map;
 
-async function resolve(filepath, filenames, root = '/') {
+async function resolve(filepath, filenames, root = path.parse(filepath).root) {
```

Comment:
or use `root = path.delimiter`?

Reply:
On windows you don't get this `\` or `\\` or `/` it depends on your volume

Reply:
Look at https://nodejs.org/api/path.html#path_path_parse_path

Reply:
Good catch! Will you also fix the tests?

Reply:
Tests are hard to fix because I want to know whether it works on linux. I will wait for https://github.com/parcel-bundler/parcel/pull/35